torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2230,16 +2230,44 @@ def error_inputs_diag(op_info, device, **kwargs):
     yield ErrorInput(SampleInput(zero_d, args=(0,)), error_type=RuntimeError,
                      error_regex="1D or 2D")
 
+
 def error_inputs_embedding(op_info, device, **kwargs):
+    weight = torch.tensor(1.0, device=device)
     indices = torch.rand(2, 2, device=device).long()
-    weights = [
-        torch.tensor(1.0, device=device),
-        torch.tensor(1.0, device=device).reshape(1, 1, 1),
-    ]
 
-    for weight in weights:
-        yield ErrorInput(SampleInput(weight, args=(indices,)), error_type=RuntimeError,
-                         error_regex="'weight' must be 2-D")
+    kwargs = (
+        dict(),
+        dict(max_norm=1.),  # tests embedding_renorm_
```

Comment:
`nn.functional.embedding` might call `embedding_renorm_`. This makes sure that branch is executed.